src/verify.py

```diff
@@ -0,0 +1,16 @@
+#
+#Write a software that verifies if a number is present in a pre-defined array.
+#Output example:
+#Insert number 3
+#The number 3 is [not] present in the array.
+
+
+N = [3, 4, 5, 1, 2, 3, 4, 9, 13, 0]
+
+print("insert number")
+v = input()
+
+if(int(v) in N): 
+    print(f"The number {v} is  present in the array.")
+else:
+    print(f"The number {v} is not present in the array.")
```

Comment:
È buona pratica evitare di lasciare parti dello script senza indentazione,
perché verrebbero eseguiti ogni volta che qualcuno importa il tuo script.
Per evitarlo, includi tutte queste righe in una funzione 'main' e aggiungi un 
```python
if __name__ == "__main__":
    main()
```
come [qui](https://www.geeksforgeeks.org/what-does-the-if-__name__-__main__-do).